roles/network-multus/tasks/deprovision.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: openshift cni config
+  set_fact:
+    cni_config: "{{ openshift_cni_config }}"
+  when: cluster=="openshift"
+
+- name: kubernetes cni config
+  set_fact:
+    cni_config: "{{ kubernetes_cni_config }}"
+  when: cluster=="kubernetes"
+
+- name: Render multus deployment yaml
+  template:
+    src: multus.yml
+    dest: /tmp/multus.yml
+
+- name: Delete multus Resources
+  command: kubectl delete -f /tmp/multus.yml --ignore-not-found
+
+- name: Render cni plugins deployment yaml
+  template:
+    src: cni-plugins.yml
+    dest: /tmp/cni-plugins.yml
+
+- name: Delete cni plugins Resources
+  command: kubectl delete -f /tmp/cni-plugins.yml --ignore-not-found
+
+- name: Render L2 plugin deployment yaml
+  template:
+    src: l2-bridge.yml
+    dest: /tmp/l2-bridge.yml
+
+- name: Delete L2 plugin Resources
+  command: kubectl delete -f /tmp/l2-bridge.yml --ignore-not-found
```

Comment:
This actually does not remove installed binaries from Nodes. Do we care?

Reply:
This doesn't remove them we know.
I am also not sure if we need to take care of that on the ansible